src/base/polyfills.js

```diff
@@ -0,0 +1,53 @@
+// Copyright 2014 Globo.com Player authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+/**
+ * Array.prototype.find
+ *
+ * Original source : https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Array/find
+ * See also : https://tc39.github.io/ecma262/#sec-array.prototype.find
+ */
+if (!Array.prototype.find) {
+  Object.defineProperty(Array.prototype, 'find', {
+    value: function(predicate) {
```

Comment:
btw, i am not sure if i can rewrite this line to `value: predicate => {`, using `=>` may bind function to this ? any tought ?